OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -413,7 +410,7 @@ public void onSessionRestarted() {
       }
    };
 
-   private static OSSessionManager sessionManager = new OSSessionManager(sessionListener);
+   @NonNull static OSSessionManager sessionManager = new OSSessionManager(sessionListener);
```

Comment:
what do you think instead keep it private and add a getter? to leave this sessionManager set responsible only to OneSignal.java

Reply:
Done